lang_python/parsing/Parser_python.mly

```diff
@@ -303,17 +304,9 @@ expr_stmt:
   | tuple(test_or_star_expr)
       { ExprStmt (tuple_expr $1) }
   (* typing-ext: *)
-  | tuple(test_or_star_expr) ":" test
+  (*| tuple(test_or_star_expr) ":" test
```

Comment:
You can't remove this code ....
It's ok to improve AST_Python.ml to help Parse_python_tree_sitter.ml, but there is no reason to remove
functionality from the existing Python parser in pfff.
In fact you have CI failures because of that.

Reply:
yeah that is actually a pretty fair point

i was a little overambitious because i was thinking about migrating primarily to tree-sitter, i think